cmake/onnxruntime_mlas.cmake

```diff
@@ -5,13 +5,29 @@ set(MLAS_ROOT ${ONNXRUNTIME_ROOT}/core/mlas)
 set(MLAS_SRC_DIR ${MLAS_ROOT}/lib)
 set(MLAS_INC_DIR ${MLAS_ROOT}/inc)
 
+# Detect SVE support in AArch64 (ARM 64-bit)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64")
+    message(STATUS "Detecting SVE support for AArch64...")
+    # Include module to check compiler flags
+    include(CheckCXXCompilerFlag)
+    # Test if the compiler supports SVE
+    check_cxx_compiler_flag("-march=armv8.2-a+sve+fp16" COMPILER_SUPPORTS_SVE)
+    if (COMPILER_SUPPORTS_SVE)
+        message(STATUS "Compiler supports SVE!")
+        add_compile_definitions(COMPILER_SUPPORTS_SVE=1) # Define COMPILER_SUPPORTS_SVE for source files
```

Comment:
`target_compile_defintions` instead of `add_compile_definitions` ? 
https://github.com/microsoft/onnxruntime/blob/main/docs/cmake_guideline.md